src/tree/EnvironmentTreeItem.ts

```diff
@@ -69,8 +70,13 @@ export class EnvironmentTreeItem extends AzureParentTreeItem implements IAzureRe
     }
 
     public get description(): string {
-        const linkedDesc: string = localize('linkedTag', '{0} (linked)', this.data.properties.sourceBranch);
-        return this.inWorkspace ? linkedDesc : this.data.properties.sourceBranch;
+        if (this.data.properties.status !== 'Ready') {
+            // if the environment isn't ready, the status has priority over displaying its linked
+            return localize('statusTag', '{0} ({1})', this.data.properties.sourceBranch, this.data.properties.status);
```

Comment:
Can you paste a screenshot of the portal? Feels like we're getting in the range of too much information on these tree items, too. Maybe we get rid of PR number? The nice thing about the status is it should be hidden for most cases...